modules/nf-core/gatk4/createsequencedictionary/tests/main.nf.test

```diff
@@ -0,0 +1,57 @@
+nextflow_process {
+
+    name "Test Process GATK4_CREATESEQUENCEDICTIONARY"
+    script "../main.nf"
+    process "GATK4_CREATESEQUENCEDICTIONARY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gatk4"
+    tag "gatk4/createsequencedictionary"
+
+    test("sarscov2 - fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("versions") },
+                { assert snapshot(file(process.out.dict.get(0).get(1)).readLines()).match() }
```

Comment:
Why not just snapshot process.out.dict?

Reply:
Like this:
`{ assert snapshot(file(process.out.dict)).match() }`?

Reply:
seems to work